src/njoy21/input/VIEWR/Card1.hpp

```diff
@@ -0,0 +1,19 @@
+class Card1 {
+public:
+  #include "njoy21/input/VIEWR/Card1/Infile.hpp"
```

Comment:
But isn't the name and the description going to be different, since it's called "infile" instead of nin?